netcon/mtclient/mtclient.h

```diff
@@ -80,6 +80,9 @@
 
 #include "ui.h"
 
+// Chat tracker
+#define CHATTRACKERNAME "tracker.pxo.nottheeye.com"
+#define CHATPORT 7170
```

Comment:
should we hardcode these values?

Reply:
This is first iteration, in future we'll extend these strings to configuration.

Reply:
yeah, registry/config file seems like a good fit for them